src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_VALUE = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        final int figureParameter = 10;
```

Comment:
```suggestion
        final int figureParameter = 10;
```
This is not a constant but a variable in the constructor. Fix it and change the name to CIRCLE_AREA